src/taylor.jl

```diff
@@ -0,0 +1,126 @@
+"""
+    series(y, x, [x0=0,] ns; name = nameof(y))
+
+Expand the variable `y` in a power series in the variable `x` around `x0` to orders `ns`.
+
+Examples
+========
+
+```julia
+julia> @variables z ϵ
+2-element Vector{Num}:
+ z
+ ϵ
+
+julia> series(z, ϵ, 2, 0:3)
+z[0] + z[1]*(-2 + ϵ) + z[2]*((-2 + ϵ)^2) + z[3]*((-2 + ϵ)^3)
+```
+"""
+function series(y, x, x0, ns; name = nameof(y))
```

Comment:
This won't work for making a taylor series for functions, like for the perturbation theory for differential equations example. Maybe adding a method like this would work:
```julia
series(coeff, x) = sum([x^(i-1)*y for (i, y) in enumerate(coeff)])
```
That could be used like
```julia
@variables t, (y(t))[1:3] epsilon
series(y, epsilon)
```

I think that function could also be used to implement all of the other use cases.
- series with $x_0 \ne 0$: `series(y .+ x0, epsilon)`
- series with first coeff zero: `series([0; y], epsilon)`

Reply:
Thanks for this suggestion. I added something similar to let the user both pass their own coefficients, or a variable to automatically define them.